src/app/producer-shelf/components/producer-shelf/producer-shelf.component.html

```diff
@@ -1,10 +1,14 @@
 <div class="wrapper">
     <div class="main">
-      <alba-action-panels [rundown]="rundown" [shelfConfiguration]="shelfConfiguration" />
+      @if (actionPanelConfigurations && actionPanelConfigurations.length > 0) {
+        <alba-action-panels [rundown]="rundown" [actionPanelConfigurations]="actionPanelConfigurations" />
+      }
       <alba-virtual-keyboard [keyBindings]="keyBindings" [keystrokes]="keystrokes" />
     </div>
 
-    <div class="sidebar">
-        <alba-static-actions [rundown]="rundown" [shelfConfiguration]="shelfConfiguration" [centerActions]="true" />
-    </div>
+      @if (staticActionIds && staticActionIds.length > 0) {
```

Comment:
If `actionPanelConfigurations` become non-nullable, then this can be simplified to just checking the length directly:

```suggestion
      @if (staticActionIds.length > 0) {
```

Reply:
Changed to the suggestion in https://github.com/tv2/alba-web-client/pull/456/commits/4fb832f0f5435f487dde1da27200b507226eb9cc